server/views/layout.jade

```diff
@@ -0,0 +1,7 @@
+doctype html
```

Comment:
@RonLek Since we are not using the backend for SSR, we won't need the views folder.